frontend/src/features/projects/dispose/ProjectDisposalSubmitted.tsx

```diff
@@ -3,20 +3,33 @@ import './ProjectDisposalSubmitted.scss';
 import * as React from 'react';
 import { Container } from 'react-bootstrap';
 import { FaRegCheckCircle } from 'react-icons/fa';
+import useKeycloakWrapper from 'hooks/useKeycloakWrapper';
+import styled from 'styled-components';
+
+const ColoredWrapper = styled.div`
+  color: #2e8540;
+`;
 
 /**
  * Display success message after all project disposal steps are completed successfully.
  */
 export function ProjectDisposalSubmitted() {
+  const keycloak = useKeycloakWrapper();
   return (
     <Container className="ProjectDisposalSubmitted">
-      <FaRegCheckCircle size={64} />
-      <h5>Application Successfully Submitted</h5>
+      <ColoredWrapper style={{ color: '#2e8540' }}>
```

Comment:
Glad to see usage of `styled-components` here - but would like to know why the double styling? Isn't `<ColoredWrapper>` defined earlier as a div with that same color?

Reply:
oops

Reply:
I'll address this in my next PR.